impl/client/src/main/doc-js/pentaho/context/IContext.jsdoc

```diff
@@ -15,14 +15,14 @@
  */
 
 /**
- * The `IContext` interface exposes contextual information of the Pentaho Web Client Platform,
- * like, for example, its
+ * The `IContext` interface allows access to the Pentaho Web Client Platform.
```

Comment:
@pentaho/millenniumfalcon I'm not sure with the change from exposes to allows access, what is your opinion?